spec/lib/efile/md/md502cr_calculator_spec.rb

```diff
@@ -35,6 +35,17 @@
               expect(instance.lines[:MD502CR_PART_M_LINE_1].value).to eq(1_750)
               expect(instance.lines[:MD502CR_PART_AA_LINE_13].value).to eq(1_750)
             end
+
+            context "deduction method is nonstandard" do
+              before do
```

Comment:
i'm curious why the test above doesn't fail since you don't explicitly stub out `deduction_method_is_standard` to equal true.

Reply:
I'm guessing the fixture that is used to calculate (`calculate_deduction_method`) whether it's standard/nonstandard returns `S`. Maybe we should be explicit and stub it out in the block above to be super clear?